src/main/java/org/igniterealtime/openfire/botz/BotzConnection.java

```diff
@@ -281,7 +281,7 @@ public void login() throws BotzSessionAlreadyExistsException {
 	 */
 	public void login(String username)
 			throws BotzSessionAlreadyExistsException, UserNotFoundException {
-		System.out.println("Firt Login");
+		Log.trace("First Login "+username);
```

Comment:
Ah, thanks for doing this! A minute further improvement would be to use SLF4J's templating API, like so:

```java
		Log.trace("First Login {}", username);
```